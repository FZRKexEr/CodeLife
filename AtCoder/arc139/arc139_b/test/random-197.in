100
19 18 19 8 6 9
4 19 5 15 20 12
65 9 14 20 15 5
91 20 15 11 15 14
8 7 8 13 5 16
3 1 8 5 6 10
14 2 6 15 9 15
38 11 19 9 9 15
93 10 10 10 17 12
55 16 19 8 14 20
62 1 5 12 18 2
70 18 14 10 15 18
3 4 11 9 7 9
55 14 9 9 12 3
52 6 3 20 4 15
84 7 2 9 12 9
31 19 20 20 20 14
68 11 12 19 9 13
26 15 11 1 16 17
10 5 1 7 16 13
34 20 16 10 10 20
55 2 6 8 16 12
95 19 8 8 4 5
23 20 3 5 15 5
13 13 12 16 13 6
73 10 17 2 14 9
52 3 4 6 18 4
59 5 5 16 8 14
43 19 7 12 8 4
85 20 6 17 14 13
25 14 11 4 19 18
78 10 15 3 2 16
67 20 12 10 2 3
43 18 17 18 15 17
98 9 12 1 6 6
32 19 2 10 10 9
13 18 7 18 6 13
43 3 19 13 8 2
52 14 6 8 2 17
35 1 12 2 4 13
83 18 10 2 1 8
52 10 2 4 14 16
34 6 7 17 9 8
78 3 7 4 1 10
74 16 19 14 14 20
8 8 1 4 16 1
59 8 7 3 14 18
56 13 11 17 4 18
46 11 4 5 12 17
8 5 8 20 18 2
16 13 5 19 9 6
14 16 6 8 5 7
30 7 7 17 15 1
25 19 15 19 13 17
80 1 14 13 6 10
88 10 2 14 7 1
1 7 5 12 10 13
92 12 11 17 1 18
81 9 15 4 17 15
75 17 15 13 19 11
77 20 15 3 18 10
9 5 7 11 13 3
32 7 8 17 1 2
83 5 5 5 6 8
12 6 16 1 18 1
29 8 14 8 8 7
14 13 7 5 20 15
12 20 8 16 2 1
47 6 9 2 14 16
15 3 4 17 6 8
92 16 13 19 6 15
82 12 12 10 12 20
7 18 10 4 8 11
58 4 7 4 18 14
50 16 12 15 10 2
48 2 19 10 18 9
69 5 15 9 19 5
93 4 17 12 1 19
69 10 20 8 1 9
72 6 5 6 12 6
40 2 2 5 3 7
73 4 4 7 8 14
11 10 17 11 6 8
78 20 17 1 10 15
73 2 13 2 8 12
92 12 7 17 18 18
57 20 4 10 18 3
78 15 12 2 14 13
89 3 14 19 3 17
36 12 13 11 15 6
22 7 3 8 6 18
64 17 1 9 10 5
70 17 11 19 5 17
4 5 7 6 10 8
21 8 16 17 15 8
90 5 10 1 15 13
99 17 10 10 14 4
14 14 11 9 11 15
34 11 18 13 1 10
59 18 7 20 1 17
