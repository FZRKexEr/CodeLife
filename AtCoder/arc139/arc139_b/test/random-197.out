9
60
35
84
16
15
39
30
138
62
48
83
27
27
67
144
234
62
26
32
60
116
20
20
13
46
52
152
36
111
37
37
40
156
50
74
121
38
55
48
14
78
41
26
88
8
107
84
58
2
37
30
38
127
110
44
12
76
99
55
56
25
8
111
12
22
25
65
84
30
92
172
28
120
38
108
75
35
78
96
60
151
17
46
76
161
52
76
92
45
26
95
58
24
93
90
66
11
16
93
